internal/cmd/configssh.go

```diff
@@ -52,11 +52,14 @@ func configSSH(configpath *string, remove *bool) func(cmd *cobra.Command, _ []st
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
 
+		usr, err := user.Current()
```

Comment:
That is going to be a problem on some systems and will prevent us from shipping a static binaries.
It has been a while, I'm looking into this to see the extent, it might have improved.

Reply:
Alright, I didn't see we were using it before already.
I checked and at least in go1.15, it does work without cgo! Which is quite neat.

Reply:
Tested with and without cgo, and tested on windows, works as expected.
Slight note on windows, the path output has double `\`.
<img width="478" alt="window ssh-config coder-cli" src="https://user-images.githubusercontent.com/521701/91771633-4ce47200-ebb1-11ea-9110-77d1bc94c8fb.PNG">

Reply:
Pushed a fix that resolves the double `\` (straight to master by accident 😰 ).